server/bin/unittests

```diff
@@ -279,10 +279,12 @@ function _setup_state {
     let res=res+$?
     mkdir -p $_testopt_sat/bin
     let res=res+$?
-    mkdir -p $_testopt/common/lib
+    mkdir -p $_testopt/common/
     let res=res+$?
     cp -a $_tdir/{unittests,pbench*} $_testopt/bin
     let res=res+$?
+    cp -a ${_tdir}/../../lib ${_testopt}/common/
```

Comment:
Rather than going "up" from `_tdir`, why not come "down" from `_gitdir`?  (And, for the record, I think there is a better way to establish `_gitdir` than what is used at lines 16-17 -- `git rev-parse --show-toplevel`.)

Reply:
Mostly because I copied and pasted without trying to trace through the whole hierarchy of environment variables that the script constructs. I'm also not entirely sure this is even necessary; I added it when my tests initially had trouble finding the DB models, after tearing out some hair I can't really afford to lose... but then I got smarter and blew away the tox virtual environment and forced it to reload, and that worked. I'd actually meant to try removing this to see if it still worked, but I forgot. (Though logically this copy is consistent with the general setup strategy and might be worth doing anyway.)